product-create-search/src/BusinessLogic/SendToKafka.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+
+namespace App\BusinessLogic;
+
+
+use Kafka\Producer;
+use Kafka\ProducerConfig;
+use Symfony\Component\Serializer\SerializerInterface;
+
+class SendToKafka implements SendToKafkaI
+{
+    private $crudMysql;
```

Comment:
The services , like this should be stateless ,
meaning : not allowed to have any variable inside like status